.azuredevops/deploy-platform-core.yaml

```diff
@@ -124,4 +124,4 @@ extends:
             resourceGroupName: $(servicesResourceGroup)
             privateEndpointDnsRecordsForResources:
               - resourceName: $(infraResourceNamePrefix)$(nc_resource_storageaccount)$(nc_instance_regionid)01
-                resourceGroupName: $(servicesResourceGroup)
+                resourceGroupName: $(servicesResourceGroup)
```

Comment:
Is anything changed in this file?

Reply:
No Ganesh, I think it is some trailing spaces.